src/scheduler/Calendar/Calendar.test.js

```diff
@@ -52,7 +52,35 @@ describe("Calendar", function() {
     expect(label).toEqual("Unavailable, Friday February 28 2020");
   });
 
-  test("Handles key nav events", async () => {
```

Comment:
This is a new test that should trigger the fail scenario we encountered in Notify, i.e. get past 11 pm, the calendar will produce a Javascript error when trying to read inexistent values of available culled time.